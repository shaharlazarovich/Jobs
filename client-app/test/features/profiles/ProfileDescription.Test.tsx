import { ReactWrapper } from "enzyme";
import React from "react";
import { findByTestAttr } from '../../common/testUtils';
import ProfileDescription from "../../../src/features/profiles/ProfileDescription";
import { setupWrapper } from "../../common/wrapper";

describe('Profile Description UniTest', () => {
    let wrapper: ReactWrapper;
    beforeEach(async () => {
        wrapper = await setupWrapper(<ProfileDescription  />);
    });
    test('renders without error', () => {
        const component = findByTestAttr(wrapper, 'component-profile-description');
        expect(component.length).toBe(1);
    });
    test('should match the snapshot', () => {
        const component = findByTestAttr(wrapper, 'component-profile-description');
        expect(component).toMatchSnapshot();
    });
})