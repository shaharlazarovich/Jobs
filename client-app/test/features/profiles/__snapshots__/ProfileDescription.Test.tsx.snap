// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Description UniTest should match the snapshot 1`] = `
<TabPane
  active={true}
  as={[Function]}
  data-test="component-profile-description"
>
  <Segment
    attached="bottom"
    className="active tab"
    data-test="component-profile-description"
  >
    <div
      className="ui bottom attached segment active tab"
      data-test="component-profile-description"
    >
      <Grid>
        <div
          className="ui grid"
        >
          <GridColumn
            width={16}
          >
            <div
              className="sixteen wide column"
            >
              <Header
                content="About Fake Username"
                floated="left"
                icon="user"
              >
                <div
                  className="ui left floated header"
                >
                  <Icon
                    as="i"
                    name="user"
                  >
                    <i
                      aria-hidden="true"
                      className="user icon"
                      onClick={[Function]}
                    />
                  </Icon>
                  <HeaderContent>
                    <div
                      className="content"
                    >
                      About Fake Username
                    </div>
                  </HeaderContent>
                </div>
              </Header>
            </div>
          </GridColumn>
          <GridColumn
            width={16}
          >
            <div
              className="sixteen wide column"
            >
              <span />
            </div>
          </GridColumn>
        </div>
      </Grid>
    </div>
  </Segment>
</TabPane>
`;
